src/Options.cc

```diff
@@ -393,6 +399,8 @@ Options parse_cmdline(int argc, char** argv) {
         {"re-level", required_argument, nullptr, 'T'},
         {"watchdog", no_argument, nullptr, 'W'},
         {"print-id", required_argument, nullptr, 'I'},
+        {"no-opt-funcs", required_argument, nullptr, '\001'},
```

Comment:
Is this passing a non-printable character to configure? I get that we've used a lot of the alphabet, but we can't have to resort to that yet, do we?

Reply:
It seemed better to me to use something clearly non-mnemonic rather than try to either find something vaguely mnemonic, or grab something printable that down the road might have a better fit to something else. But I'm fine with switching this to printable if desired

Reply:
That said, ok I'll go with '1' and '2' for these